src/coreclr/tools/AssemblyChecker/Program.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
```

Comment:
I'd like to see a header comment here that explains what this tool is, and how it is used.

Reply:
Sure, I can add it.

Reply:
If you put this in a `void PrintUsage()` method (still near the top of the file) and call it before all of the places where the tool returns 2, then you get the documentation at the top and command line help.

For "You can also ask...", add the argument so that the user doesn't have to look at the source code to figure out how.